src/Symfony/Component/Serializer/Command/DebugCommand.php

```diff
@@ -99,13 +99,20 @@ private function getAttributesData(ClassMetadataInterface $classMetadata): array
 
         foreach ($classMetadata->getAttributesMetadata() as $attributeMetadata) {
             $data[$attributeMetadata->getName()] = [
-                'groups' => $attributeMetadata->getGroups(),
-                'maxDepth' => $attributeMetadata->getMaxDepth(),
-                'serializedName' => $attributeMetadata->getSerializedName(),
-                'ignore' => $attributeMetadata->isIgnored(),
                 'normalizationContexts' => $attributeMetadata->getNormalizationContexts(),
                 'denormalizationContexts' => $attributeMetadata->getDenormalizationContexts(),
+                'groups' => $attributeMetadata->getGroups(),
+                'ignore' => $attributeMetadata->isIgnored(),
+                'maxDepth' => $attributeMetadata->getMaxDepth(),
             ];
+
+            if ($serializedName = $attributeMetadata->getSerializedName()) {
+                $data[$attributeMetadata->getName()]['serializedName'] = $serializedName;
+            }
+
+            if ($serializedPath = $attributeMetadata->getSerializedPath()) {
+                $data[$attributeMetadata->getName()]['serializedPath'] = (string) $serializedPath;
+            }
```

Comment:
It outputs information of either `SerializedName` or `SerializedPath`, because both cannot be used in combination for the same property.